bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/monopole_SpinHalf_DrellYan/cards/SpinHalf_DrellYan_3000/SpinHalf_DrellYan_3000_run_card.dat

```diff
@@ -0,0 +1,129 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update to_full                                                *
+#*********************************************************************
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+#*********************************************************************
+  100000	= nevents ! Number of unweighted events requested 
+  0	= iseed ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                3=photon from electron, 4=photon from muon          *
+#*********************************************************************
+  1	= lpp1 ! beam 1 type 
+  1	= lpp2 ! beam 2 type
+  6500.0	= ebeam1 ! beam 1 total energy in GeV
+  6500.0	= ebeam2 ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  nn23lo1	= pdlabel ! PDF set                                     
+  230000	= lhaid ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
Here you are using an old PDF set (NNPDF23_nlo_as_0119), which is different with respect to the other card.

Reply:
Ok, but why do you need an old PDF set? Can't you use the default set that you get automatically by using:
```
     'lhapdf'    = pdlabel     ! PDF set
     $DEFAULT_PDF_SETS    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
     $DEFAULT_PDF_MEMBERS = reweight_PDF
```

Reply:
I think we can, but would the distribution change? Because we want to preserve certain conditions that are useful for the monopole case.

Reply:
Yes, the distributions can be different if you change the PDF set. If you have done some studies and you are sure that 230000 is what you need, then it's fine for me. Just be aware that it is an old set (NNPDF 2.3).

Reply:
yes, we did ancient studies and this pdf is what we need